internal/boxcli/global.go

```diff
@@ -166,6 +168,25 @@ func pullGlobalCmdFunc(cmd *cobra.Command, args []string) error {
 	return box.PullGlobal(args[0])
 }
 
+func shellenvGlobalCmdFunc(cmd *cobra.Command, runInitHook bool) error {
+	path, err := ensureGlobalConfig(cmd)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	box, err := devbox.Open(path, cmd.ErrOrStderr())
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	output, err := box.PrintEnv(cmd.Context(), false, runInitHook)
```

Comment:
I think Mike added the useCache flag because he needed it in the generated wrappers code. For global, how do wrappers work @mikeland86?

If we keep the `false`, could we add a comment for the param name for readability: like `, false /* paramName*/`?